src/release_notes_workflow/releasenotes_check.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+
+
+class ReleaseNotesCheck:
```

Comment:
This still needs refactoring, https://github.com/opensearch-project/opensearch-build/pull/2438#discussion_r941468303

Reply:
Just did dB, the logic is still same, we need to segregate OS OSD and rest of all other plugins, as for the plugins release notes added as with extra `.0` please add your thoughts further @dblock.
Thank you